### PR TITLE
[python-api] add xbmc.InfoTagPicture

### PR DESCRIPTION
**Pull request by phil65**
This is the basework to allow reading imageinfo from a listitem object.
Up to now I only added 2 methods to allow fetching GPS EXIF data. Will add more once I need them. :)
@tamland @xhaggi 
I would also need an xcode sync.. Could someone step up for that? :)

### CODE REVIEW

**Comment by Razzeee** [comment]
@anaconda for xcode maybe?

**Comment by tamland** [comment]
I know you are just copy-pasting here, but since you are starting from scratch, perhaps it's a good time to make something new and better? Like a plain data class, or dict! I only added this tag crap to the ListItem API in the first place in a moment of laziness:\

**Comment by phil65** [comment]
Sounds good, but would take me too much time to get it right, sorry. Way too busy with script and skin work atm.

**Comment by anaconda** [comment]
Xcode: https://github.com/anaconda/xbmc/commit/d89fe5a28a9bbab89de633339bd44f8a0242b8a3

**Comment by phil65** [comment]
Thx @anaconda !

**Comment by ronie** [comment]
adding a new class with just the two methods 'you need' is pretty meh ;-p
would be better to implement them all imo.

**Comment by phil65** [comment]
I might add the more common ones after it is decided whether this makes it in. Adding all is out of scope, sorry, we do not even have all getters for video and music.

**Comment by ronie** [comment]
> we do not even have all getters for video and music.

that should not be an excuse ;-)

**Comment by tamland** [comment]
I'd say just use plain member variables. I will be a better api and *less* work. 1 line for definition and 1 line initialization is all you need. These getter wrappers only exists because it wasn't possible to map fields to python attributes before.

**Comment by phil65** [comment]
Updated. Instead of separate setters and getters I now make use of CPictureInfoTag::TranslateString().
Is that fine enough, @tamland?

**Comment by phil65** [comment]
Ping @tamland. :)

File: xbmc/interfaces/legacy/ListItem.h [hunk 1]
```diff
@@ -998,6 +999,22 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
       ///
       /// \ingroup python_xbmcgui_listitem
+      /// @brief \python_func{ getPictureInfoTag() }
+      ///-----------------------------------------------------------------------
+      /// Returns the PictureInfoTag for this item.
+      ///
+      /// @return     picture info tag
+      ///-----------------------------------------------------------------------
+      /// @python_v17 New method added.
```
Comment (MartijnKaijser) [inline xbmc/interfaces/legacy/ListItem.h]: v18 and some other places as well ;) 
Need to add this to some file to make  it visible for doxygen

**Comment by tamland** [comment]
> Updated. Instead of separate setters and getters I now make use of CPictureInfoTag::TranslateString().
Is that fine enough, @tamland?

are the available fields documented somewhere?

**Comment by phil65** [comment]
For now I put a link to source into the docs:
/// For a list of all available info tags, see xbmc/pictures/PictureInfoTag.cpp in code
I could also re-list them of course if wanted. I went for the way with least maintenance for now. :)

**Comment by phil65** [comment]
ping @tamland

**Comment by tamland** [comment]
not sure what im pinged for:) still two open comments. was there a question?

**Comment by phil65** [comment]
oh, that was related to whether you want me to list all available infotags in docs or whether a link to kodi source is fine enough.

**Comment by tamland** [comment]
Always nicer to have have all documentation in one place, without external links, but up to you;)

**Comment by phil65** [comment]
jenkins build this please.

**Comment by Razzeee** [comment]
rebase?

**Comment by Rechi** [comment]
@phil65 this needs a rebase

**Comment by phunkyfish** [comment]
PR closed as it appears to be abandoned by the author.

If the author would like it kept open please just request so on the PR.

**Pull request closed without merging**